news/2020-09-01-osu-idol-2020-auditions-now-open.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: post
+title: "osu!idol 2020 Auditions Now Open!"
+date: 2020-09-01 17:00:00 +0000
+---
+
+Remember that time where people were singing and our ears were ringing with joy? It's happening again. If you are interested in singing and competing to becoming osu!'s the next idol you should grab a mic and join us on our singing adventure!
+
+![](/wiki/shared/news/2020-09-01-osu-idol-2020-auditions-now-open/2020.jpg)
+
+osu!idol is the community driven singing contest where contestants participate as a solo or duet, in multiple stages consisting of various genres, showcasing their vocal skills to our panel of judges. They will evaluate their performances and determine who makes it through the stages, eventually ending up in the finals. Voting for the winner will be conducted by you, the community, to decide who will be the next osu!idol!
```

Comment:
```suggestion
osu!idol is the community driven singing contest where contestants participate as a solo or duet, in multiple stages consisting of various genres, showcasing their vocal skills to our panel of judges. The judges will evaluate their performances and determine who makes it through the stages, eventually ending up in the finals. Voting for the winner will be conducted by you, the community, to decide who will be the next osu!idol!
```

Reply:
Using "the judges" right after saying "judges" sounds too rapidly repetitive

Reply:
"**They** will evaluate **their** performances..." sounds a bit odd to me, plus *they* pronoun here can refer to *the contestants* given the previous sentence. As TPGPL suggested, *The judges* would be the safest and the least ambiguous phrasing here, from what I see.

Reply:
Fair point, though that still doesnt sound well to me. Perhaps this part needs to be worded better altogether.

_"osu!idol is the community driven singing contest where contestants showcase their vocal skills as a solo or duet, in multiple stages consisting of various genres. Our panel of judges will evaluate their performances and determine who makes it through the stages, eventually ending up in the finals. Voting for the winner will be conducted by you, the community, to decide who will be the next osu!idol!_

Reply:
yeah i think that `They` in this context refers more to contestants than judges, so the sentence might be misleading for readers

Reply:
```suggestion
osu!idol is the community driven singing contest where contestants participate as a solo or duet, in multiple stages consisting of various genres, showcasing their vocal skills to our jury. The judges will evaluate their performances and determine who makes it through the stages, eventually ending up in the finals. Voting for the winner will be conducted by you, the community, to decide who will be the next osu!idol!
```
if you dont want to repeat the `judges` word, then this suggestion might be better?

Reply:
Read my suggestion in italic above @TPGPL

Reply:
looks fine

Reply:
Will apply that then.